custom_components/tuya_local/devices/smartplug_usb_childlock.yaml

```diff
@@ -0,0 +1,95 @@
+name: Plug In WiFi Smart Socket with USB
+products:
+  - id: ynz9qmnmnsciccyc
+    name: Plug In WiFi Smart Socket with USB
+primary_entity:
+  entity: switch
+  class: outlet
+  name: USB
+  icon: "mdi:usb"
+  dps:
+    - id: 1
+      type: boolean
+      name: switch
+secondary_entities:
+  - entity: switch
+    name: AC Outlet
+    class: outlet
+    dps:
+      - id: 2
+        type: boolean
+        name: switch
+      - id: 42
+        type: string
+        name: cyclic_schedule
+      - id: 43
+        type: string
+        name: random_schedule
+      - id: 44
+        type: string
+        name: inching
+  - entity: number
+    category: config
+    name: USB Countdown Timer
+    dps:
+      - id: 9
+        type: integer
+        name: value
+        unit: min
+        range:
+          min: 0
+          max: 86400
+        mapping:
+          - scale: 60
+            step: 60
+  - entity: number
+    category: config
+    name: AC Outlet Countdown Timer
```

Comment:
Add `translation_key: timer` here and to the one above, and remove this name to use the translated name (above needs the name to disambiguate it)